raidpackage.py

```diff
@@ -0,0 +1,155 @@
+    await sent_message.add_reaction(icon_definitions.REACTION_ACCEPT)
+    await sent_message.add_reaction(icon_definitions.REACTION_CANCEL)
+    return sent_message, preorder_embed
+
+async def wait_for_order_reaction_add(payload, client, sent_message, preorder_embed, usr, order_channel):  
+
+    if str(payload.emoji) == icon_definitions.emoji_spectral_str:
+        item = "Potion of Spectral Strength"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 40, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value=item + f" x{qtyReq}", inline=False))
+            
+    if str(payload.emoji) == icon_definitions.emoji_spectral_int:
+        item = "Potion of Spectral Intellect"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 40, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value=item + f" x{qtyReq}", inline=False))
+
+    if str(payload.emoji) == icon_definitions.emoji_shaded_sharpen:
+        item = "Shaded Sharpening Stone"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 6, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value=item + f" x{qtyReq}", inline=False))
+
+    if str(payload.emoji) == icon_definitions.emoji_shaded_weightstone:
+        item = "Shaded Weightstone"
+               
+        qtyReq = await process_user_quantity_input(client, usr, item, 6, payload.user_id) 
+        if qtyReq == None:
+            return
+        else:
+            await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value=item + f" x{qtyReq}", inline=False))
+
+async def wait_for_order_reaction_remove(reaction_remove_payload, sent_message, preorder_embed):
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_spectral_str:
+        await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value='None', inline=False))
+
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_spectral_int:
+        await sent_message.edit(embed=preorder_embed.set_field_at(1, name="Potion", value='None', inline=False))
+
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_shaded_sharpen:
+        await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value='None', inline=False))
+
+    if str(reaction_remove_payload.emoji) == icon_definitions.emoji_shaded_weightstone:
+        await sent_message.edit(embed=preorder_embed.set_field_at(0, name="Weapon Enhancement", value='None', inline=False))  
+
+async def cancel_order(msg, usr_id, client):
```

Comment:
lets try to keep this convention of naming method names with underscores

Reply:
Yes, my mistake I remembered PEP as camelcase but it should be underscores